pxtlib/emitter/cloud.ts

```diff
@@ -114,25 +114,34 @@ namespace pxt.Cloud {
     }
 
     // 1h check on markdown content if not on development server
-    const MARKDOWN_EXPIRATION = pxt.BrowserUtils.isLocalHostDev() ? 1 : 1 * 60 * 60 * 1000;
+    const MARKDOWN_EXPIRATION = pxt.BrowserUtils.isLocalHostDev() ? 0 : 1 * 60 * 60 * 1000;
```

Comment:
I know this isn't a change but I'm curious - what's the 1 here for?

Reply:
the 1 in `1 * 60`? Just for expressing the time fully - 1 hour * 60 minutes * 60 seconds * 1000 ms. Not necessary at all, but I've seen it written both ways / possibly prevents silly typos if someone wanted to change it to 2 hours later.

Reply:
oo cool